ui/packages/catalog-ui-search/src/main/webapp/component/property/property.collection.view.js

```diff
@@ -287,22 +287,28 @@ module.exports = Marionette.CollectionView.extend(
     },
     /* Generates a collection view containing all properties from the metacard intersection */
     generateCollectionView: function(metacards) {
-      var propertyIntersection = this.determinePropertyIntersection(metacards)
-      var propertyArray = []
-      propertyIntersection.forEach(function(property) {
-        propertyArray.push({
-          enumFiltering: true,
-          enum: metacardDefinitions.enums[property],
-          validation: metacardDefinitions.validation[property],
-          label: properties.attributeAliases[property],
-          readOnly: metacardDefinitions.metacardTypes[property].readOnly,
-          id: property,
-          type: metacardDefinitions.metacardTypes[property].type,
-          values: {},
-          multivalued: metacardDefinitions.metacardTypes[property].multivalued,
-          required: false,
-        })
-      })
+      let propertyIntersection = this.determinePropertyIntersection(metacards)
+
+      if (
+        Array.isArray(propertyIntersection) &&
+        propertyIntersection.length === 0
+      ) {
+        propertyIntersection = metacards.flatMap(card => Object.keys(card))
```

Comment:
❓ Is this a workaround for missing metacard types?

Reply:
Yeah, I forgot to remove this during my various stabs at fixing it. I removed it.